docs/tutorial/force_calibration.rst

```diff
@@ -464,3 +464,102 @@ When working with small beads or at high laser powers, it is important to verify
 In that case, the arguments `fixed_diode` and `fixed_alpha` can be passed to :func:`~lumicks.pylake.calibrate_force()` to fix these parameters to their predetermined values.
+
+Robust fitting
+--------------
+
+So far, we have been using least-squares fitting routines for force calibration. In that case, we assume that the error in the power at each fequency is distributed according to a Gaussian distribution. The act of blocking or windowing of the power spectrum ensures that this assumption is close enough to the truth so that the fit gives accurate estimates of unknown parameters. Occasionally, the power spectrum might show a spurious noise peak. Such a peak is an outlier in the expected behavior of the spectrum and therefore interferes with the assumption of having a Gaussian error distribution. As a result, the fit is skewed. In those cases, it can be beneficial to do a so-called robust fit. When a robust fit is performed, one assumes that the probability of encountering one or multiple outliers is non-negligible. By taking this into account during fitting, the fit can be made more robust to outliers in the data, which explains the name. An example illustrates the idea. 
+
+Load a dataset of uncalibrated force sensor data of a 4.4 μm bead showing Brownian motion while being trapped. In particular, look at the `Force 2y` sensor signal::
+
+    f = lk.File("test_data/robust_fit_data.h5")
+    f2y = f.force2y
+
+First create a power spectrum without blocking or windowing, for later use. Then derive a power spectrum with blocking from the first power spectrum::
+
+    ps = lk.calculate_power_spectrum(f2y.data, sample_rate=f2y.sample_rate, num_points_per_block=1, fit_range=(10, 23e3))
+    ps_blocked = ps.downsampled_by(200)
+
+Use a passive calibration model, with hydrodynamic correction turned on. Then perform a least-squares fit and plot the result::
+
+    model = lk.PassiveCalibrationModel(4.4, temperature=25.0, hydrodynamically_correct=True)
+    fit = lk.fit_power_spectrum(ps_blocked, model)
+
+    plt.figure()
+    fit.plot()
+    plt.title(
+        f'Skewed fit: $f_c$ = {fit.results["fc"].value:.1f}, $D$ = {fit.results["D"].value:.4f}, $f_d$ = {fit.results["f_diode"].value:.1f}'
+    )
+    plt.show()
+
+.. image:: figures/force_calibration/power_spectrum_noise_peak.png
+
+Notice how the tail of the model is skewed towards the peak, in order to reduce the least-squares error. The free parameters to fit the diode filter contribution are 'abused' to reduce the error between the model and the outlier.
+
+Now do a robust fit. We do this by specifying a loss function in the function `fit_power_spectrum()`. For least-squares fitting, the loss function is `'gaussian'`, which is the default if nothing is specified. However, if we specify `'lorentzian'`, a robust fitting routine will be used instead. Because `bias_correction` and robust fitting are mutually exclusive, we need to explicitly turn it off::
```

Comment:
```suggestion
Now do a robust fit. We do this by specifying a loss function in the function :func:`~lumicks.pylake.fit_power_spectrum()`. For least-squares fitting, the loss function is `'gaussian'`, which is the default if nothing is specified. However, if we specify `'lorentzian'`, a robust fitting routine will be used instead. Because `bias_correction` and robust fitting are mutually exclusive, we need to explicitly turn it off::
```